### PR TITLE
test(forms-web-app): feature file and steps for when you can appeal page

### PR DESCRIPTION
**Pull request by davehlewis**
## Ticket Number

AS-1334

## Description of change
Feature file and tests for AS-1244

## Checklist

- [ ] Requires infrastructure changes
- [ ] If adding or remove environment variables (e.g. in `docker-compose.yaml`) then I have updated the appropriate Helm chart
- [ ] I have updated the documentation accordingly
- [X] My commit history in this PR is linear
- [X] New features have tests
- [ ] Breaking change (team conversation required)

## Important

Please do not merge from `master` (please only [rebase](https://github.com/foundry4/appeal-planning-decision/wiki/An-intro-to-Git-Rebase)). This keeps the history linear and easier to debug.

### CODE REVIEW

**Review by frank-dodion — APPROVED** [review]
I have discussed this PR with @davehlewis and he is content that the e2e currently fail.

File: e2e-tests/cypress/integration/guidance-page-when-you-can-appeal.feature [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Feature: Guidance Page - Before you Appeal
```
Comment (shilpabasu22) [inline e2e-tests/cypress/integration/guidance-page-when-you-can-appeal.feature:1]: It should be "When you can appeal"

File: e2e-tests/cypress/integration/guidance-page-when-you-can-appeal.feature [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  As a PO on the appeal service
+  I want prospective appellants to be aware of who the service is for
+  So that only those who need to appeal through the new appeal service do so
+
+  Scenario Outline: AC1: Appellant selects <page> page from the Content list
+    Given the appellant is on the when you can appeal page
```
Comment (shilpabasu22) [inline e2e-tests/cypress/integration/guidance-page-when-you-can-appeal.feature:7]: This doesn't tell me whether appellant has arrived on this page by clicking on Next button or by clicking on the link in the content list.

Comment (davehlewis) [reply e2e-tests/cypress/integration/guidance-page-when-you-can-appeal.feature:7]: Does it need to? If so this is a different test needing different acceptance criteria

Comment (shilpabasu22) [reply e2e-tests/cypress/integration/guidance-page-when-you-can-appeal.feature:7]: There is nothing wrong but an observation that this page can be accessed in 2 ways and will you be covering both the ways?

**Review by shilpabasu22 — APPROVED** [review]

**Pull request merged**